TensorMath.lua

```diff
@@ -388,6 +388,18 @@ wrap("add",
         {name=real, default=1},
         {name=Tensor}})
 
+
+wrap("csub",
+     cname("sub"),
```

Comment:
Actually no. `sub` is defined at the [lua level](https://github.com/torch/torch7/blob/master/generic/Tensor.c#L289), so it shouldn't be a problem. But maybe it can be ambiguous